src/battle/quintohit.php

```diff
@@ -41,6 +41,8 @@
 
 
 	$misschance = intval(random_int(0, 100));
+
+	$db->beginTransaction();
```

Comment:
This function doesn't exist on AdoDB connection object, we can use beginTrans

https://adodb.org/dokuwiki/doku.php?id=v5:userguide:transactions